tableauserverclient/server/endpoint/schedules_endpoint.py

```diff
@@ -70,15 +71,24 @@ def create(self, schedule_item):
     @api(version="2.8")
     def add_to_schedule(self, schedule_id, workbook=None, datasource=None,
                         task_type=TaskItem.Type.ExtractRefresh):
+        def read_warnings(response):
```

Comment:
Can this be moved out of the endpoint class? I thought I saw it in the last MR.

Reply:
Thanks for the comment! Yeah, I thought about it. The same code was in ScheduleItem in the last MR. I was not sure I should factor it out to a static function in ScheduleItem for reading warnings. It seems reading warnings should be common to all rest api calls. But I did not find a good place for this kind of "common" things. Ideas?

Reply:
Thanks for moving it to the item object